From d3ba745fed6cdbb674158dd9e383ab4dcc5127ee Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 26 Dec 2017 20:31:46 +0800
Subject: [PATCH 3145/5242] MLK-17293-7 arm: dts: imx7ulp: update cpu
 set-points

commit  6f0c822cc7c3bbc43599ea332949b7034f75e153 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to datasheet Rev-D, on B0 part, below CPU
freq needs to be supported:

500MHz for RUN mode;
720MHz for HSRUN mode.

Update opp table accordingly.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |   10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 2fb65e8..201ff09 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -43,8 +43,8 @@
 
 		operating-points = <
 			/* KHz	uV */
-			531648	1125000
-			416072	1025000
+			720000	1125000
+			500210	1025000
 		>;
 		clocks = <&clks IMX7ULP_CLK_ARM>,
 			 <&clks IMX7ULP_CLK_CORE_DIV>,
@@ -53,9 +53,11 @@
 			 <&clks IMX7ULP_CLK_HSRUN_CORE>,
 			 <&clks IMX7ULP_CLK_SPLL_PFD0>,
 			 <&clks IMX7ULP_CLK_SPLL_SEL>,
-			 <&clks IMX7ULP_CLK_FIRC>;
+			 <&clks IMX7ULP_CLK_FIRC>,
+			 <&clks IMX7ULP_CLK_SPLL>;
 		clock-names = "arm", "core_div", "sys_sel", "hsrun_sys_sel",
-			      "hsrun_core", "spll_pfd0", "spll_sel", "firc";
+			      "hsrun_core", "spll_pfd0", "spll_sel", "firc",
+			      "spll";
 		};
 	};
 
-- 
1.7.9.5

